### ISSUE TITLE
Restart of HA required when power fail or reboot of Intellicenter occurs.

### ISSUE THREAD

**Issue by kracklow** [jlvaillant/intellicenter#48]
I am having an issue constantly with anytime the IntelliCenter is rebooted or loses power I have to restart HA to get the integration talking with IC again.

**Comment by jlvaillant** [comment]
Yes, that is unfortunately a side effect of a change made to support the new IC firmware. The ‘ping’ heartbeat that was used to detect reboots and reconnect was creating other even more annoying problems. I’ll have to see if that can be addressed. In the meantime sorry for the inconvenience.

**Comment by BillyFKidney** [comment]
Is there any method to know that the connection has been lost?
